### ISSUE TITLE
Review Ports in Cassandra Chart

### ISSUE THREAD

**Issue by xtophs** [CatalystCode/fortisdeploy#224]
The Cassandra chart seems to open unnecessary ports, increasing the attack surface of the application. Cassandra ports should only be accessible from inside the cluster. Cassandra itself should not be accessible from the internet.

**Comment by c-w** [comment]
Could we also handle this in the ARM template or deployment script by adding a security group rule to block load-balancer inbound traffic on the Cassandra ports?

**Comment by xtophs** [comment]
We should check. I *believe* the chart creates a kubernetes service, which will then create the security rule to open up the port to the internet. If this is an internal-only service, we could look at creating and internal and an external service to expose to the internet only what’s needed.

HTH,
Christoph

________________________________

**Comment by erikschlegel** [comment]
we should just remove the external service, as it poses a security risk and is not needed in the stack.

**Comment by c-w** [comment]
Here is how the cluster is currently set up.

```sh
NAME:   cassandra-cluster
LAST DEPLOYED: Thu Nov 30 04:23:49 2017
NAMESPACE: cassandra
STATUS: DEPLOYED

RESOURCES:
==> v1/Service
NAME                          CLUSTER-IP  EXTERNAL-IP  PORT(S)         AGE
cassandra-cluster-cassan      None        <none>       9042/TCP        5s
cassandra-cluster-cassan-ext  10.0.27.58  <pending>    9042:31452/TCP  5s

==> v1beta1/StatefulSet
NAME                      DESIRED  CURRENT  AGE
cassandra-cluster-cassan  5        1        4s
```

Since my changes in https://github.com/CatalystCode/project-fortis-mono/commit/e3863c8209cffdcdec17409c09687306069b8ddf the external ip is no longer required: previously we used it to set up the database schema but we now do this from a docker image inside of the cluster.

This means we can now remove the external ip. The cluster ip should remain active since that's how we communicate between `spark<->cassandra<->graphql`.

**Comment by c-w** [comment]
Related to https://github.com/CatalystCode/project-fortis-pipeline/issues/240

**Comment by c-w** [comment]
Resolved by the same fix as #240.
